incident/api/tests/test_api.py

```diff
@@ -343,6 +343,8 @@ def test_result_attributes(self):
         self.assertEqual(data, {
             'id': self.category.pk,
             'title': self.category.title,
+            'slug': self.category.slug,
+            'url': "http://testserver" + self.category.url,
```

Comment:
It's interesting that you had to include "testserver" here, which feels like an artifact of how Django's test client/server is configured. This caused me to look into a few things and it seems like what you are using to determine the full url, `build_absolute_uri`, is sidestepping Wagtail's Sites settings (which is, admittedly, a system I don't love). It might make more sense here for the category `url` field to be wired up similarly to how the incident serializer does it, as a `SerializedMethodField` hooked up to this method:

https://github.com/freedomofpress/pressfreedomtracker.us/blob/060333c917636a905bedb0633ceb9378f328ef96/incident/api/serializers.py#L170-L175

Which is to say, using Wagtail to come up with the URL. I think this is slightly "safer" because it respects wagtail's settings and is consistent with the rest of the API, even if in practice it shouldn't be different from the URL constructed from the incoming request's data. 

Let me know what you think about that, though. If that change is made, then we can update the test to look at `self.category.full_url` (though this is not IMO the main reason to make this change).

Reply:
looks like it worked, thanks!